src/frontend/pages/game.tsx

```diff
@@ -161,7 +161,11 @@ export default function Home(): ReactElement {
 
         <div className={styles.gameDisplay}>
           <GameTable pointsRemaining={pointsRemaining} />
-          <div className={styles.timer}>{timeLeft}</div>
+          <div
+            className={timeLeft <= 3 ? `${styles.turnRed}` : `${styles.timer}`}
+          >
+            {timeLeft}
+          </div>
```

Comment:
You probably have to add this to the practice game page as well unless that uses a traditional Timer Component